corehq/apps/custom_data_fields/models.py

```diff
@@ -11,6 +11,9 @@
 # This list is used to grandfather in existing data, any new fields should use
 # the system prefix defined below
 SYSTEM_FIELDS = ("commtrack-supply-point",)
+# If mobile-worker is demo, this will be set to value 'demo'
+COMMCARE_USER_TYPE_KEY = 'user_type'
```

Comment:
Yeah, waiting for this PR to be merge first https://github.com/dimagi/commcare-hq/pull/10465